### ISSUE TITLE
✨ Add missing/misrouted site icons

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#8781]
## ✨ Feature

Our HTTP 4xx logs show that our clients are requesting many icon files and not finding them.  Are the icons missing?  Are the icons misrouted?  Maybe some of these aren't even legitimate - and part of a security scan?

## 🕵️ Details

/apple-touch-icon-120x120-precomposed.png
/apple-touch-icon-120x120.png
/apple-touch-icon-152x152-precomposed.png
/apple-touch-icon-152x152.png
/apple-touch-icon-precomposed.png
/apple-touch-icon.png
/talent/images/android-chrome-192x192.png
/talent/images/android-chrome-512x512.png
/talent/images/apple-touch-icon.png
/talent/images/favicon-16x16.png
/talent/images/favicon-32x32.png
/talent/images/mstile-144x144.png
/talent/images/mstile-150x150.png
/talent/images/mstile-310x150.png
/talent/images/mstile-310x310.png
/talent/images/mstile-70x70.png

## ✅ Acceptance Criteria

- [ ] Find out why these icon files are not found
- [ ] Use Figma plugin to make all the images from the original image
- [ ] Add as many as possible to the site
